### PR TITLE
Update opensnitch-hu_HU.ts

### PR DESCRIPTION
**Pull request by ovari**
Do `<translation>   …   </translation>` need to have leading and trailing spaces? https://github.com/evilsocket/opensnitch/issues/394#issuecomment-840848423

https://github.com/evilsocket/opensnitch/issues/394

### CODE REVIEW

File: ui/i18n/locales/hu_HU/opensnitch-hu_HU.ts [hunk 1]
```diff
@@ -6,656 +6,656 @@
...
         <location filename="../../../opensnitch/res/preferences.ui" line="531"/>
         <source>ebpf</source>
-        <translation type="unfinished"></translation>
+        <translation>ebpf</translation>
     </message>
     <message>
         <location filename="../../../opensnitch/res/preferences.ui" line="536"/>
         <source>audit</source>
-        <translation type="unfinished"></translation>
+        <translation>naplózás</translation>
```
Comment (gustavo-iniguez-goya) [inline ui/i18n/locales/hu_HU/opensnitch-hu_HU.ts:407]: don't translate this string, it's the name of the process monitor method (auditd).

Sorry for that, I'll make it not translatable for future releases.

**Comment by gustavo-iniguez-goya** [comment]
> Do <translation> … </translation> need to have leading and trailing spaces?

It depends on the string, but usually no.

File: ui/i18n/locales/hu_HU/opensnitch-hu_HU.ts (hunk 1 above)
Comment (ovari) [reply ui/i18n/locales/hu_HU/opensnitch-hu_HU.ts]: done

**Comment by ovari** [comment]
> > Do  …  need to have leading and trailing spaces?
> 
> It depends on the string, but usually no.

Removed leading and trailing spaces in the translation strings.

**Comment by gustavo-iniguez-goya** [comment]
Thank you very much @ovari !

**Comment by ovari** [comment]
1. Is `asocciated` spelt incorrectly in the en source strings? Should it be spelt as `associated `?
2. Should `opensnitch` be spelt as `OpenSnitch`?
3. Should `pop-up`, `popup` be standardized to be all with or without a hyphen?
https://www.microsoft.com/en-us/language/Search?&searchTerm=popup&langID=382&Source=true&productid=0
4. Should the leading and trailing spaces in the en source strings be removed?

````
<message>
        <location filename="../../../opensnitch/res/preferences.ui" line="578"/>
        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If checked, opensnitch will prompt you to allow or deny connections that don&apos;t have an asocciated PID, due to several reasons.&lt;/p&gt;&lt;p&gt;The pop-up dialog will only contain information about the network connection.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
        <translation></translation>
    </message>
````

Thank you

**Comment by ovari** [comment]
@gustavo-iniguez-goya please contact us in the future if there are new strings to translate.

Thank you once again.

**Pull request merged**